reconcile/utils/oc.py

```diff
@@ -486,12 +486,15 @@ def get_items(self, kind, **kwargs):
         resource_names = kwargs.get("resource_names")
         if resource_names:
             items = []
-            for resource_name in resource_names:
-                resource_cmd = cmd + [resource_name]
-                item = self._run_json(resource_cmd, allow_not_found=True)
-                if item:
-                    items.append(item)
-            items_list = {"items": items}
+            try:
+                items_list = self._run_json(cmd + resource_names, allow_not_found=True)
+            except ForbiddenError:
+                for resource_name in resource_names:
+                    resource_cmd = cmd + [resource_name]
+                    item = self._run_json(resource_cmd, allow_not_found=True)
```

Comment:
if trace the requests like 

```
oc get pods -o json -n some-namespace a b --v=8
```

Then it shows multiple `GET` requests to individual resources, is there a case that some requests are forbidden but some pass?

Reply:
here is an example we used to have: https://github.com/openshift/managed-cluster-config/commit/bf2fc7679e4975a31b43aa5660d6115461784300

Reply:
When there is no permission to get some resource, will it also throw `ForbiddenError` when get that individual one back?
Can we add a warning log here so we know there can be a case that enter `except ForbiddenError` then processed normally?